interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
+owner @{HOME}/.config/gtk-2.0/gtkfilechooser.ini w,
+owner @{HOME}/.config/gtk-3.0/bookmarks w,
+owner @{HOME}/.config/gtk-3.0/settings.ini w,
```

Comment:
Why are these 'w'rite and not 'r'ead? I expected applications to need to read global settings, but not set them.